app/services/hyrax/adapters/nesting_index_adapter.rb

```diff
@@ -19,11 +19,14 @@ def self.find_preservation_document_by(id:)
       def self.find_preservation_parent_ids_for(id:)
         # Not everything is guaranteed to have library_collection_ids
         # If it doesn't have it, what do we do?
-        fedora_object = ActiveFedora::Base.find(id)
-        if fedora_object.respond_to?(:member_of_collection_ids)
-          fedora_object.member_of_collection_ids
-        else
-          []
+        ActiveFedora::Base.uncached do
+          fedora_object = ActiveFedora::Base.find(id)
```

Comment:
What is the performance impact of using `uncached`?

Reply:
This is most likely an AF bug, but I'm not sure how to duplicate it lower in the stack. The indexing here is happening after the save and updating other indexing, but AF was retrieving the value from the cache and undoing the changes we just made.

Reply:
And yes I suspect it may be the reason for the problem in 1785. It was causing an error when removing a work from a collection, but it could have much broader implications as well.

Reply:
After researching #1785 bug, I realized the caching isn't an AF bug, it's just a result of the point in the process where the nested indexing was happening (mid-transaction for the update process). So removing the caching here is necessary, but not an AF caching bug.

Reply:
Is the uncached needed around just the find or just the call to member_of_collection_ids or both?

Reply:
It is only needed on the find. I am changing this.